### PR TITLE
Avoid session path below graph when selecting a session

### PR DESCRIPTION
**Pull request by 3v0k4**

### CODE REVIEW

File: app/javascript/angular/code/calculateBounds.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const pixelToLength = (pixels, zoom) => pixels * Math.pow(2, -zoom);
```
Comment (3v0k4) [inline app/javascript/angular/code/calculateBounds.js]: mv?

File: app/javascript/angular/code/calculateBounds.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+const PIXELS = 1000;
```
Comment (3v0k4) [inline app/javascript/angular/code/calculateBounds.js]: rename

File: app/javascript/angular/code/calculateBounds.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { pixelsToLength } from "./mapsUtils.js";
+
+// The graph is ~220px high. For whatever reason the OFFSET_IN_PIXEL
+// must be 1000 to not have the session path covered by the graph.
+const OFFSET_IN_PIXEL = 1000;
+
+export const calculateBounds_ = pixelsToLength => (sensors, sessions, zoom) => {
```
Comment (gitjul) [inline app/javascript/angular/code/calculateBounds.js]: why pass the function as a parameter?

File: app/javascript/angular/code/calculateBounds.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import { pixelsToLength } from "./mapsUtils.js";
+
+// The graph is ~220px high. For whatever reason the OFFSET_IN_PIXEL
+// must be 1000 to not have the session path covered by the graph.
+const OFFSET_IN_PIXEL = 1000;
```
Comment (gitjul) [inline app/javascript/angular/code/calculateBounds.js]: 🤔 will take a look when/if I style anything there

File: app/javascript/angular/code/calculateBounds.js (hunk 3 above)
Comment (3v0k4) [reply app/javascript/angular/code/calculateBounds.js]: Testing reasons

File: app/javascript/angular/code/calculateBounds.js (hunk 4 above)
Comment (3v0k4) [reply app/javascript/angular/code/calculateBounds.js]: No worries it's gone ;)

**Pull request merged**